v2.9.0 (#851)
DataDog/datadog-ci: README.md

To release a new version of `datadog-ci`:

1. Create a new branch for the version upgrade.

> 2. Update the `package.json` version, commit the change `vX.X.X` and tag it with `git tag vX.X.X`. Depending on the nature of the changes introduced. You may refer to [Semantic Versioning](https://semver.org/#summary) to determine what level to increment.

Reviewer: ```suggestion
2. Update the `package.json` version, commit the change `vX.X.X` and tag it with `git tag vX.X.X`, based on the version you are upgrading to. You may refer to [Semantic Versioning](https://semver.org/#summary) to determine what level to increment.
```

Reviewer: Let me know if that's not correct.
